### DISCUSSION TITLE
tab_

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#617] [ОШБК!] [answered]
При создании кнопки с tab_ она хардкорно скрывается, как сделать ее видимой?

**Comment by erpic-ru** [comment, marked as answer]
Сделать рядом кнопку без tab_ которая будет нажимать кнопку с tab_

**Reply by dmbrtlt** [reply]
спасибо

**Comment by LostAlly** [comment]
А что делает такая кнопка? Не смог в документации найти.

**Reply by erpic-ru** [reply]
https://ru.docs.totum.online/tables#type-tab-in-cyclea
